lib/p2p/Pool.ts

```diff
@@ -118,10 +118,14 @@ class Pool extends EventEmitter {
     this.bindNodeList();
 
     this.nodes.load().then(() => {
```

Comment:
Maybe refactor this to use async/await. It will also help making the code simpler with the new conditions.

Reply:
I would prefer to as well, but this was deliberate as per https://github.com/ExchangeUnion/xud/pull/389 so that this `init` method can resolve before all the connections to peers resolve.